cabal-testsuite/PackageTests/LinkerOptions/T7339/setup.test.hs

```diff
@@ -17,51 +17,51 @@ import Distribution.System
 import Test.Cabal.Prelude
 
 
-main = setupTest $ do
+main = do
+  setupTest $ do
+    skipIfNoSharedLibraries
+    skipUnlessGhcVersion ">= 8.4"
 
-  skipIfWindows
```

Comment:
For the reviewer: Same here, I only changed `windows -> noSharedLibraries`. Rest is indentation.